### PR TITLE
Add CONTRIBUTING.md

### LINKED ISSUE #120: Add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by eliazonta**
Added CONTRIBUTING.md based on PyG 
resolves #120

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -1 +1,121 @@
...
+Once you finish implementing a feature or bug-fix, please send a PR to https://github.com/pyg-team/pytorch-frame.
+
+Your PR will be merged after one or more rounds of reviews by the [pyg-team](https://github.com/pyg-team).
+
+
+## Developing PyTorch Frame
+
+To develop PyTorch Frame on your machine, here are some tips:
+
+1. Ensure that you are running on one of the two latest PyTorch releases (*e.g.*, `2.0.0`):
```
Comment (weihua916) [inline .github/CONTRIBUTING.md:19]: 2.1.0

**Review by weihua916 — APPROVED** [review]
LGTM, thanks!

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (akihironitta) [reply .github/CONTRIBUTING.md:19]:
```suggestion
1. Ensure that you are running on one of the supported PyTorch versions (*e.g.*, `2.1.0`):
```

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -1 +1,121 @@
...
+   ```bash
+   pip uninstall pytorch_frame
+   ```
+
+3. Clone a copy of PyTorch Frame from source:
+
+   ```bash
+   git clone https://github.com/pyg-team/pytorch-frame.git
+   cd pytorch-frame
+   ```
```
Comment (akihironitta) [inline .github/CONTRIBUTING.md:38]:
```suggestion
3. Fork and clone the PyTorch Frame repository:

   ```bash
   git clone https://github.com/<your_username>/pytorch-frame.git
   cd pytorch-frame
   ```
```

**Review by akihironitta — APPROVED** [review]

**Pull request merged**